trie/secure_trie.go

```diff
@@ -85,6 +87,21 @@ func (t *SecureTrie) TryGetNode(path []byte) ([]byte, int, error) {
 	return t.trie.TryGetNode(path)
 }
 
+// TryUpdate account will abstract the write of an account to the
+// secure trie.
+func (t *SecureTrie) TryUpdateAccount(key []byte, acc *types.StateAccount) error {
```

Comment:
This method uses a pointer `acc *types.StateAccount`, but `trie.TryUpdateAccount` uses `account types.StateAccount`. Is there any particular reason not to be consistent?